### PR TITLE
[stable/gatekeeper] Add Gatekeeper chart

### PR DESCRIPTION
**Pull request by jimmidyson**
#### What this PR does / why we need it:

Adds [Gatekeeper](https://github.com/open-policy-agent/gatekeeper/) chart to install Gatekeeper,
the Policy Controller for Kubernetes.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/gatekeeper/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Gatekeeper
+
+[Gatekeeper](https://github.com/open-policy-agent/gatekeeper/), the Policy Controller for
+Kubernetes.
+
+## Prerequisites
+
+- Kubernetes 1.15 (or newer) for validating and mutating webhook admission
```
Comment (ritazh) [inline stable/gatekeeper/README.md:8]: Kubernetes 1.15 is not a must. Other than supporting `DELETE`. Is that why it's required here?

**Review by ritazh — COMMENTED** [review]
Thank you for making this! 🎉

File: stable/gatekeeper/README.md (hunk 1 above)
Comment (jimmidyson) [reply stable/gatekeeper/README.md:8]: No actually it's not required I guess, it's just all I've tested against. I can change this to 1.9 safely I think.

**Comment by jimmidyson** [comment]
Ah sure. I didn't know that. I'll close this and try to get it into gatekeeper repo. Thanks!

**Comment by ritazh** [comment]
@jimmidyson Do you mind creating a PR for this in the gatekeeper repo? Thanks!

**Pull request closed without merging**